docs/internal.md

```diff
@@ -118,6 +118,7 @@ The volume label is "cidata", as defined by [cloud-init NoCloud](https://cloudin
 - `LIMA_CIDATA_MNT`: the mount point of the disk. `/mnt/lima-cidata`.
 - `LIMA_CIDATA_USER`: the user name string
 - `LIMA_CIDATA_UID`: the numeric UID
+- `LIMA_CIDATA_HOST_HOME`: the mount point of `~` or empty if not mounted
```

Comment:
Maybe `LIMA_CIDATA_HOME_MOUNTPOINT` is a better name?

Reply:
Seems reasonable.